tasks/hyrax_dev.rake

```diff
@@ -30,6 +30,25 @@ task :i18n_sorter do
   end
 end
 
+namespace :engine_cart do
+  # This generate task should only add its action to an existing generate task
+  raise 'engine_cart:generate task should already be defined' unless Rake::Task.task_defined?('engine_cart:generate')
```

Comment:
When would this `raise` happen? When this rake file is loaded? If so, will that break production apps when this code is evaluated? https://github.com/samvera/hyrax/blob/master/Rakefile#L20